### DISCUSSION TITLE
Consulta sobre conflictos en PR

### DISCUSSION THREAD

**Discussion by lucianotgm12** [frlp-utn-ingsoft/catedra#150] [Q&A] [answered]
Buenas noches, tengo una consulta sobre un PR que realizó un miembro del grupo, pero no estoy comprendiendo el conflicto que muestra GitHub con dos archivos de tests entre main y la branch en la que trabajó para incluir los tests de ruff que tenía asignados.
![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/167022332/4bcb76a1-ed85-4741-aa4b-2e76eebcffbc)
![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/167022332/c8be956e-b296-4166-bcdb-590c50b52ab5)

Cuando lo revisé no vi inconvenientes en el código, por eso me extraña que haya reportado que existen conflictos. No se termina de entender cómo lo podemos resolver.

**Comment by RodrigoJacznik** [comment]
Buenas, es un conflicto de merge. Lo tenes que resolver antes de aceptar el pr

**Reply by lucianotgm12** [reply]
Si por supuesto, pero el tema está en que no sé cómo resolver los conflictos en estos casos, no entiendo cómo puedo solucionarlo para poder avanzar con el resto del trabajo práctico.

**Reply by RodrigoJacznik** [reply, marked as answer]
Creo que en el primer conflicto te tenes que quedar con la 2da parte y en el 2do conflicto quedarte con las 2
